app/browser/tabMessageBox.js

```diff
@@ -45,10 +45,20 @@ const tabMessageBox = {
     })
 
     process.on('window-prompt', (webContents, extraData, title, message, defaultPromptText,
-          shouldDisplaySuppressCheckbox, isBeforeUnloadDialog, isReload, muonCb) => {
-      console.warn('window.prompt is not supported yet')
-      let suppress = false
-      muonCb(null, '', suppress)
+         shouldDisplaySuppressCheckbox, isBeforeUnloadDialog, isReload, muonCb) => {
+      const tabId = webContents.getId()
+      const detail = {
+        message,
+        title,
+        buttons: ['ok', 'cancel'],
```

Comment:
Don't forget to add translation for this two buttons

Reply:
Can do! Did you want me to replace the other locations this text appears as well?

E.g.
https://github.com/AlexRobinson-/browser-laptop/blob/ef1cb6caf42f1a79c71715e20bb2f450e58d74b6/app/browser/tabMessageBox.js#L24

https://github.com/AlexRobinson-/browser-laptop/blob/ef1cb6caf42f1a79c71715e20bb2f450e58d74b6/app/browser/tabMessageBox.js#L38

Reply:
@AlexRobinson- that would be great 😄 👍